### ISSUE TITLE
Subset for ADE20K

### ISSUE THREAD

**Issue by Animatory** [boschresearch/OASIS#7]
Thank you for your work! 
Could you prove a list of images and classes that you used for training on ADE20K?

**Comment by SushkoVadim** [comment]
Hi,

We simply used the whole ADE20k dataset. It contains 20210 and 2000 images in the training and validation sets respectively.
We did not filter out any classes either, using all the 150 semantic classes (+1 for the unlabeled class).

To use our code, one can simply download `train/val` splits from the [official page](http://sceneparsing.csail.mit.edu/), and then point flag `--dataroot` to the folder with the dataset. 

You may also have a look at our [dataloader](https://github.com/boschresearch/OASIS/blob/master/dataloaders/Ade20kDataset.py), where we collect all images from `$dataroot/images/$data_split/` and all label annotations from `$dataroot/annotations/$data_split/.` 

Hope this answers your question.

**Comment by Animatory** [comment]
I have downloaded this dataset recently and now it has a different structure from you implemented in your dataset class (there is not split into `images` and `annotations`  folders). Also, it has 25574 train images and 3150 semantic classes (150 classes cover about 90% of the dataset). So I thought that you selected only a subset, but the dataset just evolving.

If I deal with the training I may create a PR with a new format dataset class if you don't mind

**Comment by SushkoVadim** [comment]
I have just downloaded the dataset from the official dataset page, clicking on `Scene Parsing -> [train/val (922MB)]`.
The dataset structure has not changed from what we used in our experiments.
Could you try this [link](http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip)?

**Comment by SushkoVadim** [comment]
Anyway, a contribution with a link to a newer ADE20k version is more than welcome;)

**Comment by Animatory** [comment]
I suppose this is an older version, but thanks. I downloaded new version here [http://groups.csail.mit.edu/vision/datasets/ADE20K/](link). It is possible to get it only after filling the form and getting approval.
